pkg/controller/controller.go

```diff
@@ -130,29 +137,32 @@ func (c *Controller) handleGroup(oldGroup, newGroup *userv1.Group) {
 
 		// For each added user, check if a project exists with the same name as the user
 		for _, user := range addedUsers {
+			roleBindingName := fmt.Sprintf("%s-edit", user)
+
 			// Check if a project exists with the same name as the user
-			project, err := c.projectClient.ProjectV1().Projects().Get(context.Background(), user, metav1.GetOptions{})
+			_, err := c.projectClient.ProjectV1().Projects().Get(context.Background(), user, metav1.GetOptions{})
 			if err != nil {
 				if errors.IsNotFound(err) {
 					klog.Infof("Project %s not found for user %s", user, user)
-					// TODO: Create project logic could go here
-					project := &projectv1.Project{
-						ObjectMeta: metav1.ObjectMeta{
-							Name: user,
-						},
-					}
-					_, err = c.projectClient.ProjectV1().Projects().Create(context.Background(), project, metav1.CreateOptions{})
-					if err != nil {
-						klog.Errorf("Error creating project for user %s: %v", user, err)
-					} else {
-						klog.Infof("Successfully created project %s for user %s", user, user)
-					}
+					_ = c.createUserProject(user)
 				} else {
 					// Just log the error for now
 					klog.Errorf("Error checking if project exists for user %s: %v", user, err)
 				}
 			} else {
-				klog.Infof("Project %s already exists for user %s", project.Name, user)
+				klog.Infof("Project %s already exists for user %s", user, user)
+
+				_, err := c.rbacClient.RoleBindings(user).Get(context.Background(), roleBindingName, metav1.GetOptions{})
```

Comment:
Should we just call `createRoleBinding` here?

Reply:
@johnmcollier I've moved the existent checks into `createUserProject` and `createRoleBinding` functions (commit: https://github.com/redhat-ai-dev/rosa-namespace-provisioner/pull/4/commits/2423d433a2f31b8b861b0d1d22e5e43c3f384a39) so we can simplify the logic here.